packages/vuetify/src/labs/VDateInput/VDateInput.tsx

```diff
@@ -95,7 +100,21 @@ export const VDateInput = genericComponent<VDateInputSlots>()({
       return adapter.isValid(model.value) ? adapter.format(adapter.date(model.value), 'keyboardDate') : ''
     })
 
+    const inputmode = computed(() => {
+      if (!mobile.value) return undefined
+      if (isEditingInput.value) return 'text'
+
+      return 'none'
+    })
+
     const isInteractive = computed(() => !props.disabled && !props.readonly)
+    const isReadonly = computed(() => !(mobile.value && isEditingInput.value) && props.readonly)
+
+    watch(menu, val => {
+      if (val) return
+
+      isEditingInput.value = false
+    })
```

Comment:
This is redundant because we already have `onUpdateMenuModel` which does the same thing